fuzz.c

```diff
@@ -63,6 +63,11 @@ static void fuzz_getFileName(honggfuzz_t * hfuzz, char *fileName)
              (int)getpid(), (unsigned long int)tv.tv_sec, util_rnd64(), hfuzz->fileExtn);
 }
 
+void fuzz_getExtension(const char *file_name,char *extension)  
+{  
+    extension = strrchr(file_name, '.' );   
```

Comment:
This will simply update a local copy of the 'extension' var, you probably want to pass char **extension and do

*extension = 

or simply return char*.